plane/source/docs/qhover-mode.rst

```diff
@@ -0,0 +1,142 @@
+.. _qhover-mode:
+
+===========
+QHOVER Mode
+===========
+
+In QHOVER mode, QuadPlane maintains a consistent altitude while
+allowing roll, pitch, and yaw to be controlled normally. This page
+contains important information about using and tuning alt hold.
+
+Overview
+========
+
+When altitude hold mode (aka QHOVER) is selected, the throttle is
+automatically controlled to maintain the current altitude. Roll, Pitch
```

Comment:
I think it should say that throttle controls climb rate, with mid-throttle meaning hold height